tee-worker/scripts/litentry/release/release.sh

```diff
@@ -0,0 +1,336 @@
+#!/bin/bash
+
+export LOG="log/worker0.log"
+export TARGET_DIRECTORY="tmp/w0"
+export PARACHAIN_SOURCE=$(cd .. && pwd)
+export WORKER_DIR=$(pwd)
+
+echo "Source Directory for Parachain: ${PARACHAIN_SOURCE}"
+echo "Worker Directory: ${WORKER_DIR}"
+
+function print_divider() {
+  echo "------------------------------------------------------------"
+}
+
+# TODO: Set Pipe fail if any of the commands fail
+generate_service_file() {
+  if [ "$#" -ne 4 ]; then
+    echo "Usage: generate_service_file <service_name> <description> <command> <working_directory>"
+    return 1
+  fi
+
+  local service_name="$1"
+  local description="$2"
+  local command="$3"
+  local working_directory="$4"
+
+  local service_template="[Unit]
+Description=${description}
+
+[Service]
+ExecStart=${command}
+WorkingDirectory=${working_directory}
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+"
+
+  local service_filename="${service_name}.service"
+  echo "$service_template" > "$service_filename"
+  echo "Service file \"${service_filename}\" generated successfully."
+}
+
+generate_service_file_for_worker() {
+ local service_name="$1"
+ local description="$2"
+ local command="$3"
+ local working_directory="$4"
+
+ local service_template="[Unit]
+ Description=${description}
+ After=network.target
+
+ [Service]
+ ExecStart=${command}
+ Restart=always
+ Environment='RUST_LOG=info,integritee_service=debug,ws=warn,sp_io=error,substrate_api_client=warn,itc_parentchain_light_client=info,jsonrpsee_ws_client=warn,jsonrpsee_ws_server=warn,enclave_runtime=debug,ita_stf=debug,its_rpc_handler=warn,itc_rpc_client=warn,its_consensus_common=debug,its_state=warn,its_consensus_aura=warn,aura*=warn,its_consensus_slots=warn,itp_attestation_handler=debug,http_req=debug,lc_mock_server=warn,itc_rest_client=debug,lc_credentials=debug,lc_identity_verification=debug,lc_stf_task_receiver=debug,lc_stf_task_sender=debug,lc_data_providers=debug,itp_top_pool=debug,itc_parentchain_indirect_calls_executor=debug'
+ StandardOutput=file:${WORKER_DIR}/${LOG}
+ StandardError=inherit
+
+ [Install]
+ WantedBy=default.target
+ "
+   local service_filename="${service_name}.service"
+   echo "$service_template" > "$service_filename"
+   echo "Service file \"${service_filename}\" generated successfully."
+}
+
+generate_worker_service_file() {
+  local command="/bin/bash -c  'cd ${WORKER_DIR}/tmp/w0 && ./integritee-service --clean-reset -P 2000 -w 2001 -r 3443 -h 4545 --running-mode mock --enable-mock-server --parentchain-start-block 0 run --skip-ra --dev'"
```

Comment:
same applies to `--skip-ra --dev` which only exists for development env (internal).

Generally I still favor to put the config into a json file, like the current setup :(

Reply:
Updated, Using a JSON config ✅